CHANGES.rst

```diff
@@ -4,12 +4,16 @@ Changelog
 5.0.5 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Make tox run lint, refactor plone versions, fix import sorting for plonetheme.task package.
+  [balavec]
 
 
 5.0.4 (2019-11-28)
 ------------------
 
+- Make code-analysis actually run on code, fix flake8 issues
```

Comment:
That should go probably up to 5.0.5

Reply:
no thats release like this was a more general fix, the new fixes make sure that we use the generated tox config and let them run on generated packages, to find problems in the generated packages earlier.